I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/01-definitions.rpy", line 278, in script
    $ ui.interact(suppress_overlay=True, suppress_window=True) ##这是关键
  File "game/01-definitions.rpy", line 278, in <module>
    $ ui.interact(suppress_overlay=True, suppress_window=True) ##这是关键
TypeError: 'Dissolve' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/01-definitions.rpy", line 278, in script
    $ ui.interact(suppress_overlay=True, suppress_window=True) ##这是关键
  File "D:\renpy-8.0.3-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy-8.0.3-sdk\renpy\python.py", line 1061, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/01-definitions.rpy", line 278, in <module>
    $ ui.interact(suppress_overlay=True, suppress_window=True) ##这是关键
  File "D:\renpy-8.0.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy-8.0.3-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\renpy-8.0.3-sdk\renpy\display\core.py", line 3752, in interact_core
    trans = self.ongoing_transition[None](
TypeError: 'Dissolve' object is not callable

Windows-10-10.0.17763 AMD64
Ren'Py 8.0.3.22090809
untitled 0.2
Sun Apr 16 18:40:43 2023
